packages/core/tests/part.test.js

```diff
@@ -273,33 +273,137 @@ it("Should generate the part transforms", () => {
   expect(part.attributes.list['transform-origin'][0]).to.equal('10.5 39')
 });
 
-it("Should set the part grain", () => {
+it("Should add the part cut", () => {
+  let pattern = new freesewing.Pattern();
+  let part = new pattern.Part();
+  part.addCut(4, 'fabric', true)
+  expect(part.cut.materials.fabric.cut).to.equal(4)
+  expect(part.cut.materials.fabric.identical).to.equal(true)
+  part.addCut()
+  expect(part.cut.materials.fabric.cut).to.equal(2)
+  expect(part.cut.materials.fabric.identical).to.equal(false)
+});
+
+it("Should generate an error if cut is not a number", () => {
+  let pattern = new freesewing.Pattern();
+  let part = new pattern.Part();
+  part.addCut('a', 'fabric', true)
+  expect(pattern.events.error.length).to.equal(1)
+  expect(pattern.events.error[0]).to.equal('Tried to set cut to a value that is not a number')
+});
+
+it("Should generate an error if material is not a string", () => {
+  let pattern = new freesewing.Pattern();
+  let part = new pattern.Part();
+  part.addCut(3, 4)
+  expect(pattern.events.error.length).to.equal(1)
+  expect(pattern.events.error[0]).to.equal('Tried to set material to a value that is not a string')
+});
+
+it("Should generate an error when removing a material that is not set (through addCut)", () => {
+  let pattern = new freesewing.Pattern();
+  let part = new pattern.Part();
+  part.addCut(4, 'fabric', true)
+  part.addCut(false, 'lining')
+  expect(pattern.events.error.length).to.equal(1)
+  expect(pattern.events.error[0]).to.equal('Tried to remove a material that is not set')
+});
+
+it("Should remove the part cut through addCut", () => {
+  let pattern = new freesewing.Pattern();
+  let part = new pattern.Part();
+  part.addCut(4, 'fabric', true)
+  part.addCut(false, 'fabric')
+  expect(typeof part.cut.materials.fabric).to.equal('undefined')
+});
+
+it("Should remove the part cut through removeCut", () => {
+  let pattern = new freesewing.Pattern();
+  let part = new pattern.Part();
+  part.addCut(4, 'fabric', true)
+  part.removeCut('fabric')
+  expect(typeof part.cut.materials.fabric).to.equal('undefined')
+});
+
+it("Should generate an error when removing a material that is not set (through removeCut)", () => {
+  let pattern = new freesewing.Pattern();
+  let part = new pattern.Part();
+  part.addCut(4, 'fabric', true)
+  part.removeCut('lining')
+  expect(pattern.events.error.length).to.equal(1)
+  expect(pattern.events.error[0]).to.equal('Tried to remove a material that is not set')
+});
+
+it("Should generate an error when removing a material that is not set (through removeCut)", () => {
```

Comment:
this text doesn't match the test

Reply:
Not sure what you mean, but it does. And if it doesn't the test wouldn't pass :thinking:

Reply:
What I meant was that either the description of what this test is testing for is incorrect, or this test is a duplicate of the test before it, which has the exact same description. In reading a second time, I lean more towards duplicate, but my first reading I leaned more toward that it might be testing something different

Reply:
Ah ok, I see. These used to return different warnings depending on whether you try to remove a non-existing material type, or pass in a non-string.

However, since `removeCut` now calls `addCut` under the hood, as you so correctly suggested, the warning is the same.

Still, I think it's good to test both scenarios just in case we touch `addCut` later so I'll leave them bot in, but I'll update the title